doc/api/util.md

```diff
@@ -995,11 +995,32 @@ throw an error.
 ### `util.promisify.custom`
 <!-- YAML
 added: v8.0.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/31672
+    description: This is now defined as a shared symbol.
 -->
 
 * {symbol} that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
+In addition to being accessible through `util.promisify.custom`, this
+symbol is [registered globally][global symbol registry] and can be
+accessed in any environment as `Symbol.for('nodejs.util.promisify.custom')`.
```

Comment:
It should be noted that any code that uses `Symbol.for('nodejs.util.promisify.custom')` syntax rather than the `util.promisify.custom` accessor will not work properly on older versions of Node.js so if compatibility with older versions is important, use of the accessor is preferred.

Reply:
That will be visible in the changelog.

Reply:
I'd prefer a more explicit comment on it.

Reply:
I&nbsp;recommend doing&nbsp;that in&nbsp;its&nbsp;own&nbsp;PR, as&nbsp;it&nbsp;will affect&nbsp;the&nbsp;documentation of&nbsp;[`util.inspect.custom`](https://nodejs.org/api/util.html#util_util_inspect_custom) (and&nbsp;possibly other&nbsp;symbols)&nbsp;as&nbsp;well.